### PR TITLE
Update ares help and README files

### PR DESCRIPTION
**Pull request by ssuminahn**
:Release Notes:
Update ares help and README files

:Detailed Notes:
- Update ares help file
- Fix typos and update compatibility in README file

:Testing Performed:
1. Pass unit test on ose and auto
2. Check ares -h
3. Check README file

:Issues Addressed:
[PLAT-131090] Release CLI v2.0.3 for OSE 2.9.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +96,7 @@ The following key-value pairs are the default configurations for the test.
 - Test ares-generate command.
 
     ``` shell
-    $ jasmine --device=webOS --ip=192.168.0.12 --port=24 spec/jsSpec/arse-generate.js
+    $ jasmine --device=webOS --ip=192.168.0.12 --port=24 spec/jsSpec/ares-generate.js
```
Comment (YoungeunKim) [inline README.md:99]: Good 👍

**Comment by ssuminahn** [comment]
code :+1: 
verify :+1:

File: README.md [hunk 2]
```diff
@@ -16,14 +16,10 @@ $ npm install -g @webosose/ares-cli
 
 ## Compatibility
 
-Our release cycle is independent of webOS OSE and Auto. We follow semver and here is the compatibility table:
+Our release cycle is independent of webOS OSE and Auto. 
+We recommend using the latest CLI. The latest CLI is compatible with the latest webOS OSE and webOS Auto"
```
Comment (YoungeunKim) [inline README.md:20]: Please update "the latest webOS OSE and Auto", cause the upper sentence is 
"Our release cycle is independent of webOS OSE and Auto."

**Review by YoungeunKim — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (YoungeunKim) [reply README.md:20]: Minor changes needed.
Code 👎

Comment (ssuminahn) [reply README.md]: Done

**Review by ssuminahn — COMMENTED** [review]
code :+1:

**Pull request merged**